lib/big_brother/cluster.rb

```diff
@@ -301,7 +301,16 @@ def _add_maintenance_node
     end
 
     def _check_downpage
-      total_health = @nodes.collect{ |n| n.weight || 0 }.reduce(:+)
+      local_health = @nodes.collect{ |n| n.weight || 0 }.reduce(:+)
```

Comment:
If you change this line to:
```
local_health = @nodes.collect{ |n| n.weight || 0 }.reduce(:+) || 0
```
you can remove the nil check below. Same for the next line.